lib/raidenclient/RaidenClient.ts

```diff
@@ -110,7 +112,7 @@ class RaidenClient extends BaseClient {
    */
   private sendRequest = (endpoint: string, method: string, payload?: object): Promise<http.IncomingMessage> => {
     return new Promise((resolve, reject) => {
-      if (this.isDisabled()) {
+      if (!this.isConnected()) {
         reject(errors.RAIDEN_IS_DISABLED);
```

Comment:
I think we should have two checks with two separate errors here as well, one if raiden is disabled and one if it's disconnected.

Reply:
done

Reply:
I don't see the separate checks here, could you make sure you put those in? Although since this is a Raiden class and this PR is focusing on lnd, if you want to leave this as is that's fine too and we'll get back to Raiden in a future PR.